src/illixr.hpp

```diff
@@ -30,14 +30,14 @@
             setenv(#ENV, "False", false);        \
         }                                        \
     }
-#define _STR(y)      #y
-#define STRINGIZE(x) _STR(x)
-#define GET_LONG(NAME, ENV)                                                    \
-    if (options.count(#NAME)) {                                                \
-        setenv(#ENV, std::to_string(options[#NAME].as<long>()).c_str(), true); \
-    } else if (config[#NAME]) {                                                \
-        setenv(#ENV, std::to_string(config[#NAME].as<long>()).c_str(), true);  \
-    }
+#define STR(y)       #y
+#define STRINGIZE(x) STR(x)
+//#define GET_LONG(NAME, ENV)                                                    \
```

Comment:
Nit: Build step throws a multi-line comment warning. Suggest using a `/* */` comment for the multi-line define `#define GET_LONG...`.

Reply:
I will fix this in the final merge